xbmc/video/VideoDatabase.cpp

```diff
@@ -2432,6 +2503,9 @@ int CVideoDatabase::SetDetailsForEpisode(const std::string& strFilenameAndPath,
     AddActorLinksToItem(idEpisode, MediaTypeEpisode, "director", details.m_director);
     AddActorLinksToItem(idEpisode, MediaTypeEpisode, "writer", details.m_writingCredits);
 
+    // add ratings
+    details.m_iIndexRating = AddRatings(idEpisode, MediaTypeEpisode, details.m_rating, details.m_strDefaultRating);
```

Comment:
Hm I'm not sure if we should do this here (same for tvshows etc). We also don't do it for movie sets.